src/Transport/Result.php

```diff
@@ -2,16 +2,18 @@
 
 declare(strict_types=1);
 
-namespace Sentry;
+namespace Sentry\Transport;
+
+use Sentry\Event;
 
 /**
  * This class contains the details of the sending operation of an event, e.g.
  * if it was sent successfully or if it was skipped because of some reason.
  */
-final class Response
```

Comment:
What's the point of removing `final` here? This class is a value object and it doesn't need to be extended by third parties